src/controllers/list.js

```diff
@@ -851,6 +851,10 @@ class ItemsView {
             setTitle(null);
             getItem(params).then(function (item) {
                 setTitle(item);
+                if (item.Type == 'Genre') {
```

Comment:
Just for my own understanding, why is this change needed?

Reply:
Without the ParentId, media from other libraries matching the genre will be queued as well.